best_bank_as/views.py

```diff
@@ -48,6 +50,36 @@ def get_details(request: HttpRequest, pk: int) -> HttpResponse:
     balance = account.get_balance()
     transactions = account.get_transactions()
 
+    print("Acc ID:", pk)
+    print("Details:", account)
+    print("Balance:", balance)
+
     context = {"account": account, "balance": balance, "transactions": transactions}
 
     return render(request, "best_bank_as/accounts/details.html", context)
+
+
+@login_required()
+def transfer_money(request: HttpRequest) -> HttpResponse:
+    """View to transfer money from account to account."""
+    if request.method == "POST":
+        form = TransferForm(data=request.POST, user=request.user)
+        if form.is_valid():
+            source_account = form.cleaned_data["source_account"]
+            destination_account = form.cleaned_data["destination_account"]
+            destination_account_instance = Account.objects.get(
+                account_number=destination_account
+            )
+
+            amount = form.cleaned_data["amount"]
+
+            Ledger.transfer(source_account, destination_account_instance, amount)
+
+            return redirect("best_bank_as:index")
+        else:
+            print(form.errors)
+    else:
+        form = TransferForm(user=request.user)
+    return render(
+        request, "best_bank_as/handle_funds/transfer-money.html", {"form": form}
+    )
```

Comment:
```suggestion
@login_required()
def transfer_money(request: HttpRequest) -> HttpResponse:
    """View to transfer money from account to account."""

    if request.method != "POST":
        return render(
            request,
            "best_bank_as/handle_funds/transfer-money.html",
            {"form": TransferForm(user=request.user)},
        )

    form = TransferForm(data=request.POST, user=request.user)
    if not form.is_valid():
        return render(
            request,
            "best_bank_as/handle_funds/transfer-money.html",
            {"form": form},
        )

    source_account = form.cleaned_data["source_account"]
    destination_account = form.cleaned_data["destination_account"]
    destination_account_instance = Account.objects.get(
        account_number=destination_account
    )

    amount = form.cleaned_data["amount"]
    Ledger.transfer(source_account, destination_account_instance, amount)

    return redirect("best_bank_as:index")
```

Looks good, but does this work?
It is generally better to avoid nesting if-statements when possible